### DISCUSSION TITLE
What is the OpenCoder prompt_template

### DISCUSSION THREAD

**Discussion by rudiservo** [TabbyML/tabby#3414] [Q&A] [answered]
Anyone knows what is the prompt_template for OpenCoder?

It is suposed to be based on llama but llama2 "<PRE> {prefix} <SUF>{suffix} <MID>" but does not work
Codestral "[SUFFIX]{suffix}[PREFIX]{prefix}" works somewhat but always adding a lot of spaces.

**Comment by zwpaper** [comment, marked as answer]
Hi @rudiservo, if the OpenCoder you're referring to is from infly.Ai, unfortunately, it does NOT support the FIM task, so you cannot use it for completion.

Ref: https://huggingface.co/infly/OpenCoder-8B-Instruct/discussions/2#67304d50de572a8535e3d20b
